x-pack/plugins/endpoint/scripts/resolver_generator.ts

```diff
@@ -44,6 +45,12 @@ async function main() {
       default: 'metrics-endpoint-default-1',
       type: 'string',
     },
+    policyIndex: {
+      alias: 'pi',
+      describe: 'index to store host policy in',
+      default: 'metrics-endpoint-policy-1',
```

Comment:
If we end up going with a new index I think it'll be `metrics-endpoint.policy-default-1`. This format is in line with what the beats ship as well.